task/yq/0.1/yq.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: yq
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.24.1"
+    tekton.dev/tags: yq
+    tekton.dev/displayName: "YQ replace"
+spec:
+  description: >-
+    This task can be used to replace fields in YAML files. For example for altering helm charts on GitOps repos.
+  workspaces:
+    - name: shared-workspace
+      description: A workspace that contains the fetched git repository.
+  params:
+    - name: file
+      type: string
+    - name: expression
+      type: string
```

Comment:
can we have a brief description of these params (although not required but asking so that CLI users can view this description)?

Reply:
Added!